src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AwaitExpressionSpiller.cs

```diff
@@ -478,7 +488,7 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
             var result = ArrayBuilder<BoundExpression>.GetInstance();
             for (int i = 0; i <= lastSpill; i++)
             {
-                var refKind = (!refKinds.IsDefaultOrEmpty && refKinds.Length > i && refKinds[i] != RefKind.None) ? RefKind.Ref : RefKind.None;
+                var refKind = refKinds.IsDefaultOrEmpty? RefKind.None: refKinds[i];
```

Comment:
Autoformat did add spaces.

---
In reply to: [127508455](https://github.com/dotnet/roslyn/pull/20883#discussion_r127508455) [](ancestors = 127508455,127502096)